.github/workflows/project_management.yml

```diff
@@ -1,374 +1,374 @@
-# ---
+---
 
-# name: "Project Board Automation"
+name: "Project Board Automation"
 
-# "on":
-#   pull_request_target:
-#     branches: [main]
-#     types: [synchronize, opened, reopened, labeled, unlabeled, ready_for_review, review_requested, converted_to_draft, closed]
-#   pull_request_review:
-#     types: [submitted]
+"on":
+  pull_request:
```

Comment:
I think we should use pull_request_target:

Reference: 
- https://nathandavison.com/blog/github-actions-and-the-threat-of-malicious-pull-requests - very valuable blog post
- https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/